style.css

```diff
@@ -730,3 +728,118 @@ li {
   cursor: pointer;
   color: #ff6b00;
 }
+
+.contain {
+  padding-top: 10px;
+}
+
+.section-form {
+  padding-bottom: 20px;
+}
+
+.section-form .btn {
+  margin-top: 15px;
+}
+
+.form-control {
+  width: 320px;
+  padding-left: 15px;
+  border: 0;
+  border-bottom: 1px solid hsl(15, 5%, 85%);
+  border-radius: 0;
+  background-color: transparent;
+  box-shadow: none;
+  outline: 0 none transparent;
+  -webkit-appearance: none;
+  min-height: 50px;
+}
+
+textarea {
+  margin-top: 40px;
+  min-height: 50px;
+  resize: none;
+}
+
+.form-control:focus {
+  box-shadow: none;
+  border-bottom-color: #e1e1e1;
+}
+
+.project8 {
+  color: #fff;
+  font-weight: 800;
+  margin-top: 8px;
+  gap: 10px;
+  width: 120px;
+  height: 45px;
+  background: #ff6b00;
+  border-color: #ff6b00;
+  border-bottom: #ff6b00;
+  border-right: #ff6b00;
+  border-left: #ff6b00;
+  border-top: #ff6b00;
+}
+
+.project8:hover {
+  background-color: #f0bf9c;
+  transition-duration: 0.05s;
+  cursor: pointer;
+  color: #ff6b00;
+}
+
+.myform {
+  text-align: center;
+}
```

Comment:
- [ ] Although your style is great, It's among the primary requirements to apply `flexbox` to style your form. This is to make sure you can apply a different form of alignment in your design.